NYTPhotoViewer/Resources/NYTPhotoViewerCloseButtonX.png

```diff
@@ -0,0 +1 @@
+../NYTPhotoViewer.bundle/NYTPhotoViewerCloseButtonX.png
```

Comment:
I see these PNGs are duplicated, in the new .bundle and in the location (`Resources` folder).  is that necessary?

Reply:
Unfortunately, yes. One would think the resources could just be copied from the `.bundle` (since it is just a folder) but without the `Resources` folder SwiftPM will not bother to create a bundle (or at least it skips creating the `SWIFTPM_MODULE_BUNDLE` macro that is used [here](https://github.com/nytimes/NYTPhotoViewer/pull/333/files#diff-87ecad7785f3b25d98adfa3fa646064fb4b37497cce2d8ab5b09404e887af6bfR16) to access the SwiftPM bundle).

I did give this a shot because it made plenty of sense to me: I removed the `Resources` folder and its symlinked files and changed the respective `copy` lines of the package manifest but then I got a build error in Xcode that `SWIFTPM_MODULE_BUNDLE` was not defined. Reverting my change allowed me to build successfully again.

There are a few places where the symlinks in this PR could be removed up by making changes to the project structure as a whole to align more closely with what SwiftPM expects out-of-box (the Resources folder being one of those) but my approach here was to leave as much alone as possible; adding symlinks meant I did not need to change the project at large.

Reply:
Actually, I want to call out explicitly something I've said but not made really front and center so far; although GitHub makes it look like I copied a bunch of files, they are all actually symlinks. I'm still not over-the-moon about them, but at least this has not created a situation where these files need to be modified/deleted/etc. in multiple places.

Reply:
I've been looking into this, hoping to find a way to avoid adding the complication of maintaining symlinks to the resources in the .bundle. Your commit history tells me you traveled a similar road vis-a-vis the headers. I was hoping something like this would work:
```
            resources: [
                .copy("NYTPhotoViewer.bundle/NYTPhotoViewerCloseButtonX.png")
```
but found [this bit of code](https://github.com/apple/swift-package-manager/blob/main/Sources/PackageLoading/TargetSourcesBuilder.swift#L369-L382) causes NYTPhotoViewer.bundle to be added to the sources, not the png. 

I'm still wrapping my head around the internals of SPM, but I'm thinking the final answer will be to reorganize the resources and includes of NYTPhotoViewer so that SPM can be supported without symlinks.

Reply:
That makes sense. If it were mine to maintain, I would prefer the project reorganization over the symlinks as well. 

Let me know if there’s anything else I can do to help out. I’m no expert, but I do have a bit of history with SPM at this point.

Reply:
I asked about this on the Swift forums, see https://forums.swift.org/t/package-resources-from-a-directory-named-with-an-extension/43423